releases/3.0.0/artifacts.json

```diff
@@ -0,0 +1,152 @@
+{
+  "3.0.0": {
+    "holoscan": {
+      "debian-version": "3.0.0.0-1",
+      "wheel-version": "3.0.0",
+      "base-images": {
+        "dgpu": "nvcr.io/nvidia/cuda:12.6.0-runtime-ubuntu22.04",
+        "igpu": "nvcr.io/nvidia/tensorrt:24.08-py3-igpu"
+      },
+      "build-images": {
+        "igpu": {
+          "jetson-agx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.0.0.0-igpu",
+          "igx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.0.0.0-igpu",
+          "sbsa": "nvcr.io/nvstaging/holoscan/holoscan:v3.0.0.0-igpu"
+        },
+        "dgpu": {
+          "x64-workstation": "nvcr.io/nvstaging/holoscan/holoscan:v3.0.0.0-dgpu",
+          "igx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.0.0.0-dgpu",
+          "sbsa": "nvcr.io/nvstaging/holoscan/holoscan:v3.0.0.0-dgpu",
+          "clara-agx-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.0.0.0-dgpu"
```

Comment:
This one is not listed in the PR description. Still needed?

Reply:
I will address this in #24